tbtadm/controller.cpp

```diff
@@ -122,9 +119,15 @@ std::string read(const fs::path& path)
     return content;
 }
 
+/* Trim right characters */
+std::string rtrim(const std::string& str, const std::string& chars = " \n\r")
```

Comment:
The point is that those are the only chars we actually need to trim in some places? Because if this tries to be more generic, it misses `\t`, for example, and the better way to do it is with `isspace` (and then using something like `std::find_if_not` with reverse iterator instead of `std::string::find*` variants).

Reply:
@westeri I suppose the only characters we need to trim from sysfs is a newline character, right? Otherwise something is going wrong.

Reply:
Right, typically sysfs attributes include terminating "\n" so that's probably something you may need to remove.